app/assets/stylesheets/components/curriculum_details.scss

```diff
@@ -51,10 +50,13 @@
 
       h3 {
         margin-bottom: .2em;
-      }
-
-      p {
-        font-size: .9em;
+        margin-top: 20px;
+        font-size: 18px;
+        font-weight: bold;
+        letter-spacing: 0.2px;
+        font-family: OpenSans;
```

Comment:
Hey @sdg123 👋 is the h3 not opensans by default? we have all the heading tags set to open sans in the global.scss file I believe.